Story/Episode 5/cm_052415_05.csv

```diff
@@ -1,8 +1,8 @@
 skit01#0,"""/ci1 2 Quickly! This way!"""
-skit02#0,"""/ci1 2 Huh, we're safe?"""
+skit02#0,"""/ci1 2 W-we're safe?"""
 skit03#0,"""/ci1 2 Yes, we're saved because of them! Thank you so much."""
 skit04#0,"""/ci1 2 Two need evacuation, pronto! There might be people left, so let's hurry!"""
 skit01#1,"""Quickly! This way!"""
-skit02#1,"""Huh, we're safe?"""
+skit02#1,"""W-we're safe?"""
```

Comment:
We usually try to avoid this in text when possible.

Reply:
Noted. You can close the PR then, I'll just revert my commit later.